net/wfb-ng/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wfb-ng
+PKG_VERSION:=24.9.1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng.git
+PKG_SOURCE_VERSION:=a41cd378e7e23dd094d6c50f26c4faa8e64e5954
+PKG_MIRROR_HASH:=362b3d4262e78ae0885cc25cea34e553dc29e1941be47724ece796bd35220d88
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
This is already default IMHO.

Reply:
fixed

Reply:
@BKPepe I've made all required changes. Could you merge my PR?